src/main/java/com/dev/cinema/model/dto/MovieSessionDtoRequest.java

```diff
@@ -0,0 +1,40 @@
+package com.dev.cinema.model.dto;
+
+public class MovieSessionDtoRequest {
+    private Long movieId;
+    private String movieTitle;
```

Comment:
do we need this field? is it really informative for creating MovieSession object?

Reply:
I'm pretty sure that we need Id's. But If say for real, I can't answer you for sure do we need this field or not and will it be informative. So I'll just remove it. O__o

Reply:
we need ids in this case, and we can get the whole entity by id, so there is no need to pass the title as well